### PR TITLE
T732: network separation test was added

### PR DESCRIPTION
**Pull request by Tolsi**

### CODE REVIEW

File: src/it/scala/com/wavesplatform/it/Docker.scala [hunk 1]
```diff
@@ -89,17 +93,28 @@ class Docker(suiteConfig: Config = ConfigFactory.empty) extends AutoCloseable wi
       extractHostPort(ports, networkPort),
       networkPort.toInt,
       containerInfo.networkSettings().ipAddress(),
+      containerInfo.networkSettings().networks().asScala("waves").ipAddress(),
       containerId,
       extractHostPort(ports, matcherApiPort))
     nodes += containerId -> nodeInfo
 
-    new Node(actualConfig, nodeInfo, http, timer)
+    println(client.listNetworks().asScala.map(_.scope()))
```
Comment (phearnot) [inline src/it/scala/com/wavesplatform/it/Docker.scala]: remove `println`

File: src/it/scala/com/wavesplatform/it/ValidChainGenerationSpec.scala [hunk 2]
```diff
@@ -57,8 +57,9 @@ class ValidChainGenerationSpec(allNodes: Seq[Node]) extends FreeSpec with ScalaF
       _ <- processRequests(generateRequests(b))
       height <- traverse(allNodes)(_.height).map(_.max)
       _ <- traverse(allNodes)(_.waitForHeight(height + 40)) // wait a little longer to prevent rollbacks...
-      blocks <- traverse(allNodes)(_.waitForHeight(height + 35)) // ...before requesting actual blocks
-    } yield blocks, 5.minutes)
+      _ <- traverse(allNodes)(_.waitForHeight(height + 35)) // ...before requesting actual blocks
+      blocks <- traverse(allNodes)(_.blockAt(height + 35)) // ...before requesting actual blocks
+    } yield blocks.map(_.signature), 5.minutes)
```
Comment (phearnot) [inline src/it/scala/com/wavesplatform/it/ValidChainGenerationSpec.scala]: Remove duplicate comments.

File: src/it/container/start-waves.sh [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-WAVES_IP=`ifconfig eth0 | awk '/inet addr/ {gsub("addr:", "", $2); print $2}'`
+WAVES_IP=`ifconfig eth1 | awk '/inet addr/ {gsub("addr:", "", $2); print $2}'`
```
Comment (phearnot) [inline src/it/container/start-waves.sh]: Why did you change this?

Comment (Tolsi) [inline src/it/container/start-waves.sh]: Because from now the connection between the nodes I use another interface - the network interface, which I created myself and disconnect it during the test. And node api communicates by ip of eth0.
